### DISCUSSION TITLE
Lighting cannot cast shadows from the model onto the geojson layer

### DISCUSSION THREAD

**Discussion by sanbai0** [visgl/deck.gl#6924] [Q&A] [answered]
I use geojson layer to draw the road surface and scenographlayer to draw the building;
But when I add light, the shadow of the building cannot be cast on the road;

And I found that the two seem not to be affected by each other in terms of lighting (when I stretch the height of the road beyond the building to cover the building, the lighting effect of the building will not disappear)

![image](https://user-images.githubusercontent.com/37434602/169257942-56966be9-2dcd-4724-8c6a-f637b8a33125.png)

Is this normal? What can I do to make the shadow of the building cast on the road?

**Comment by Pessimistress** [comment, marked as answer]
There is no shadow unless you explicitly enable it. See https://deck.gl/docs/api-reference/core/lighting-effect
